components/Tasks/TasksList/TasksList.tsx

```diff
@@ -0,0 +1,23 @@
+import { useRouter } from 'next/router';
+
+export default function TasksList({
+  tasks,
+}: {
+  tasks: string[];
+}) {
+  const { pathname } = useRouter();
+
+  return (
+    <ul className="tasks__list">
```

Comment:
The BEM approach has been violated

the component is called "TasksList", and therefore for the first tag in the component, the class will have a name that is consonant with the name of the component "tasks-list".
the rest of the classes will get starts with  "tasks-list" and __example will be added